src/styles/page.scss

```
@@ -0,0 +1,9 @@
+.page {
+  font-family: "Roboto", Helvetica, sans-serif;
+  display: flex;
+  justify-content: center;
+}
+
+.page__body {
+  margin: 0;
+}
```

you need to use scss syntax, please check other places 
```suggestion
.page {
  font-family: "Roboto", Helvetica, sans-serif;
  display: flex;
  justify-content: center;
  
  &__body {
     margin: 0;
  }
}
```